projects/storefrontapp-e2e-cypress/cypress/integration/smoke/merchandising-carousel.e2e-spec.ts

```diff
@@ -0,0 +1,12 @@
+context('Merchandising Carousel', () => {
```

Comment:
this shouldn't be in the smoke test. It would be in the regression folder.
This is like a unit test, and not an e2e test, where we test a user's flow

Reply:
I've moved the test to the regression area. We definitely need a test like this as its more of an integration test, rather than a unit test, to ensure that the carousel is using our strategy service and the payload is being translated correctly. Whether we need an actual e2e test, I'm not convinced, as the carousel will display regardless of metrics (user behaviour)

Reply:
as discussed in the meeting, we will most likely drop this e2e test because we would need a dedicated cds instance for Spartacus.

Reply:
I've removed this